wagtailio/static/js/main.js

```diff
@@ -16,4 +18,6 @@ document.addEventListener('DOMContentLoaded', () => {
     initComponent(GetStartedMenu);
     initComponent(SiteSearch);
     initComponent(FooterMenuColumn);
+    initComponent(FeatureIndex);
```

Comment:
Suggestion: Could we pull this out into its own bundle and add it to the page template instead?

Reply:
Makes sense. I'll add a ticket due to time constraints : )